src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.prl.enums.manageorders;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.prl.enums.CustomEnumSerializer;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum CaseTransferOptionsEnum {
+    @JsonProperty("theHighCourt")
+    theHighCourt("theHighCourt", "The high court"),
+    @JsonProperty("countyCourt")
+    countyCourt("countyCourt", "County court"),
+    @JsonProperty("familyProceedingCourt")
+    familyProceedingCourt("familyProceedingCourt", "Family proceeding court");
```

Comment:
As per figma, text should be 'Family proceedings court'.

Reply:
Corrected as per figma